features/dd-sdk-android-logs/src/main/kotlin/com/datadog/android/log/internal/logger/DatadogLogHandler.kt

```diff
@@ -44,8 +45,13 @@ internal class DatadogLogHandler(
         }
 
         val resolvedTimeStamp = timestamp ?: System.currentTimeMillis()
+        var combinedAttributes = attributes
```

Comment:
since we are going to mutate it, let's make a copy

```suggestion
        val combinedAttributes = attributes.toMutableMap()
```